mirte_telemetrix/scripts/ROS_telemetrix_aio_api.py

```diff
@@ -342,15 +342,28 @@ def __init__(self, board, sensor):
         self.max_freq = -1
         self.last_publish_value = Encoder()
         self.speed_count = 0
+        self.last_step = 0
 
     def get_data(self, req):
         return GetEncoderResponse(self.last_publish_value.value)
 
     async def start(self):
         if board_mapping.get_mcu() == "pico":
-            await self.board.set_pin_mode_encoder(
-                self.pins["pin"], 0, self.publish_data, False
-            )
+            if "A" in self.pins or "trigger" in self.pins or "digital" in self.pins:
```

Comment:
As discussed, it is better to have this fixed in the config file instead of over here.